frontend/controller/actions/group.js

```diff
@@ -456,6 +450,9 @@ export default (sbp('sbp/selectors/register', {
             await sbp('chelonia/contract/wait', params.contractID)
             await sbp('gi.actions/group/inviteAccept', {
               ...omit(params, ['options', 'action', 'hooks', 'encryptionKeyId', 'signingKeyId']),
+              data: {
+                reference: rootState[userID].groups[params.contractID].hash
+              },
```

Comment:
Hmm.. Is this a new feature? Or .... why wasn't this already here?

Reply:
New feature, kind of. This is there to make leaving and joining groups work correctly by adding a 'reference'. This way, when, e.g., you leave a group, you only leave if the reference value matches. This prevents an old 'leave' action from making you leave a group.

Reply:
Could you add a comment explaining this somewhere?